packages/styling/src/styles/theme.ts

```diff
@@ -194,6 +194,8 @@ function _makeSemanticColorsFromPalette(p: IPalette, isInverted: boolean, depCom
     disabledSubtext: p.neutralQuaternary,
     disabledBodyText: p.neutralTertiary,
     disabledBodySubtext: p.neutralTertiaryAlt,
+    disabledBorder: p.neutralTertiaryAlt,
+    disabledBorderAlt: p.themeLight,
```

Comment:
Having a themed color assigned to a semantic slot that has disabled word in it is very confusing. Even the description:
> The alternate disabled border color for elements that are active e.g. slider borders.

disabled border ... active elements. Is the element disabled or active. Is this semantic slot going to be reused by any other  component or it's very specific to only `Slider`?
@phkuo any thoughts?